RAMP/ramp.py

```diff
@@ -96,8 +97,12 @@ def unpack(bundle):
 @click.option('--dependencies', callback=jsons_str_tuple_to_jsons_tuple, help='list of module dependencies: <name, uri, sha256>')
 @click.option('--output', '-o', default='module.zip', help='output file name')
 @click.option('--print-filename-only', '-P', is_flag=True, default=False, help="Print package path, but don't generate file")
+@click.option('--packname-file', default=None, help="Print package name to the file")
 @click.option('--verbose', '-v', is_flag=True, default=False, help='verbose mode: print the resulting metadata')
+@click.option('--debug', is_flag=True, default=False, help='Print interaction with Redis. Implies --verbose.')
 def pack(module, *args, **kwargs):
+    config.debug = kwargs['debug']
```

Comment:
I think its better to create a config class and pass it then create a file with globals

Reply:
It's much more accessible that way vs moving a Config class around. IMHO global Config class is not that bad, as it is... configuration.

Reply:
Not saying you have to pass it, just thought it will be nicer to have a class config with all the variables and have a single global variable called config. not a must though ...

Reply:
Yep, sounds reasonable. I'll arrange it.

Reply:
I've tried a few approaches and the current one (config-as-a-module) seems to be the most user-friendly, although I prefer the config-as-a-class better. I found no elegant way of defining a Config class, and accessing a global instance of that class with a simple `config` expression.